src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/model/BuildMemory.java

```diff
@@ -1057,15 +1061,15 @@ public String toString() {
              * If the provided project is the same as this entry is referencing.
              * It does so by checking the fullName for equality.
              *
-             * @param other the other project to check
+             * @param otherName the other project name to check
              * @return true if so.
              * @see #getProject()
              */
-            public boolean isProject(Job other) {
-                if (this.project != null && other != null) {
-                    return this.project.equals(other.getFullName());
+            public boolean isProject(String otherName) {
```

Comment:
the original signature can still be kept and deprecated, don't want to break any scripts or other unknown integrations if we can avoid it.